src/main/java/nextstep/sessions/domain/SessionRegistration.java

```diff
@@ -29,18 +33,18 @@ public void recruitEnd() {
     this.status = SessionStatus.END;
   }
 
-  public void enrolment(NsUser user) {
-    if (users.size() > capacity) {
+  public void enrolment(Session session, NsUser user) {
+    if (students.size() > capacity) {
```

Comment:
```suggestion
    if (students.overFull(capacity)) {
```
위와 같이  메시지를 보낼 수도 있지 않을까?